test/src/main/java/org/apache/accumulo/test/functional/BulkLoadIT.java

```diff
@@ -231,6 +232,11 @@ private void testBulkFile(boolean offline, boolean usePlan) throws Exception {
         hashes.put(endRow, new HashSet<>());
       }
 
+      // Add a junk file, should be ignored
+      FSDataOutputStream out = fs.create(new Path(dir + "/junk"));
```

Comment:
I think that the `Paths.get(dir, "junk")` syntax might be a bit cleaner than the concatenation. Not a bug, but probably better, in that it's less platform dependent on the OS path separator char, and therefore a good habit to get into.